### DISCUSSION TITLE
有人跟我一样出了这个成就吗？

### DISCUSSION THREAD

**Discussion by xingyuehao** [share121/inter-knot#3205] [其他]
洗脸成就：哈士奇那连出两次骨头。
![屏幕截图 2024-08-19 234546](https://github.com/user-attachments/assets/d977886f-9816-4710-be73-da1819f17b10)
<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >

**Comment by March-777th** [comment]
谢邀，今天刚出<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
